Source/WebKit/Modules/iOS_Private.modulemap

```diff
@@ -0,0 +1,3138 @@
+    header "WKWindowFeaturesRef.h"
+    export *
+  }
+
+  explicit module WebAllowDenyPolicyListener {
+    header "WebAllowDenyPolicyListener.h"
+    export *
+  }
+
+  explicit module WebApplicationCache {
+    header "WebApplicationCache.h"
+    export *
+  }
+
+  explicit module WebArchive {
+    header "WebArchive.h"
+    export *
+  }
+
+  explicit module WebAutocapitalizeTypes {
+    header "WebAutocapitalizeTypes.h"
+    export *
+  }
+
+  explicit module WebBackForwardList {
+    header "WebBackForwardList.h"
+    export *
+  }
+
+  explicit module WebBackForwardListPrivate {
+    header "WebBackForwardListPrivate.h"
+    export *
+  }
+
+  explicit module WebCache {
+    header "WebCache.h"
+    export *
+  }
+
+  explicit module WebCaretChangeListener {
+    header "WebCaretChangeListener.h"
+    export *
+  }
+
+  explicit module WebCoreStatistics {
+    header "WebCoreStatistics.h"
+    export *
+  }
+
+  explicit module WebCoreThread {
+    header "WebCoreThread.h"
+    export *
+  }
+
+  explicit module WebCoreThreadMessage {
+    header "WebCoreThreadMessage.h"
+    export *
+  }
+
+  explicit module WebCoreThreadRun {
+    header "WebCoreThreadRun.h"
+    export *
+  }
+
+  explicit module WebCreateFragmentInternal {
+    header "WebCreateFragmentInternal.h"
+    export *
+  }
+
+  explicit module WebDOMOperations {
+    header "WebDOMOperations.h"
+    export *
+  }
+
+  explicit module WebDOMOperationsPrivate {
+    header "WebDOMOperationsPrivate.h"
+    export *
+  }
+
+  explicit module WebDataSource {
+    header "WebDataSource.h"
+    export *
+  }
+
+  explicit module WebDataSourcePrivate {
+    header "WebDataSourcePrivate.h"
+    export *
+  }
+
+  explicit module WebDatabaseManagerPrivate {
+    header "WebDatabaseManagerPrivate.h"
+    export *
+  }
+
+  explicit module WebDatabaseQuotaManager {
+    header "WebDatabaseQuotaManager.h"
+    export *
+  }
+
+  explicit module WebDefaultPolicyDelegate {
+    header "WebDefaultPolicyDelegate.h"
+    export *
+  }
+
+  explicit module WebDeviceOrientation {
+    header "WebDeviceOrientation.h"
+    export *
+  }
+
+  explicit module WebDeviceOrientationProvider {
+    header "WebDeviceOrientationProvider.h"
+    export *
+  }
+
+  explicit module WebDeviceOrientationProviderMock {
+    header "WebDeviceOrientationProviderMock.h"
+    export *
+  }
+
+  explicit module WebDocument {
+    header "WebDocument.h"
+    export *
+  }
+
+  explicit module WebDocumentPrivate {
+    header "WebDocumentPrivate.h"
+    export *
+  }
+
+  explicit module WebDownload {
+    header "WebDownload.h"
+    export *
+  }
+
+  explicit module WebEditingDelegate {
+    header "WebEditingDelegate.h"
+    export *
+  }
+
+  explicit module WebEditingDelegatePrivate {
+    header "WebEditingDelegatePrivate.h"
+    export *
+  }
+
+  explicit module WebEvent {
+    header "WebEvent.h"
+    export *
+  }
+
+#if defined(ENABLE_IOS_TOUCH_EVENTS) || defined(WK_IS_CATALYST)
```

Comment:
What does WK_IS_CATALYST accomplish? In wtf/PlatformEnableCocoa.h, ENABLE_IOS_TOUCH_EVENTS is defined as `PLATFORM(IOS_FAMILY) && USE(APPLE_INTERNAL_SDK)`, and IOS_FAMILY includes Catalyst.

Reply:
Hmm, I'm not sure, but an Apple internal build somehow had `ENABLE_IOS_TOUCH_EVENTS =0` for Mac Catalyst. 🤔

Reply:
I think this is a bug in the makefile. `FEATURE_AND_PLATFORM_DEFINES` is set up using `:=`, and at that point `TARGET_TRIPLE_FLAGS` isn't defined, so it expands to nothing. I think it works on other platforms due to clang paying attention to the `SDKROOT` environment variable, although `SDK_FLAGS` has the same problem. I'm going to try reordering those declarations and see if that fixes it. Although by fixing that bug, who knows what chaos will ensue...

Reply:
OK I _think_ this should do it finally, running another Mac Catalyst test 🤞

Reply:
Everything's passing finally! 🥳